setup.py

```diff
@@ -23,6 +23,7 @@
     "pyyaml",
     "cloudpickle>=1.6.0,<=2.2.1,!=2.1.0,!=2.2.0;python_version<'3.11'",
     "cloudpickle==2.2.1;python_version~='3.11'",  # backend only supports cloudpickle 2.2.1 + python 3.11 at the moment
+    "packaging",
```

Comment:
do we need an approval from legal team?

Reply:
we also need update in the recipe: https://github.com/snowflakedb/snowpark-python/blob/main/recipe/meta.yaml